apps/webcert/src/utils/certificate/getClientValidationErrors.test.ts

```diff
@@ -186,6 +186,37 @@ describe('Validation based on value', () => {
 })
 
 describe('Validation based on config', () => {
+  describe(`${ConfigTypes.UE_CHECKBOX_DATE_RANGE_LIST}`, () => {
```

Comment:
Hmm, a bit confused about this test. `fakeDateElement` does not return a `UE_CHECKBOX_DATE_RANGE_LIST` it returns a `UE_DATE`. Not sure how `config.minDate` gives this result when `isDateBeforeLimit` is looking a `config.min`.
😕

Reply:
My bad copy paste error